certified-connectors/Farsight Security/README.md

```diff
@@ -0,0 +1,31 @@
+# Farsight DNSDB Connector
+
+Farsight Security DNSDB® is the world’s largest DNS intelligence database that provides a fact-based view of the configuration of the global Internet infrastructure. DNSDB leverages Farsight’s Security Information Exchange (SIE) data-sharing platform and is engineered and operated by leading DNS experts. Farsight collects, filters, and verifies Passive DNS data from its global sensor array. DNSDB is the highest-quality and most comprehensive DNS intelligence data service of its kind.
```

Comment:
@RamboV Adding copy rights and register symbols in any connector file (including readme.md and intro.md) is not allowed, Our legal team will not let us certify with either the copyright or registered trademarks anywhere in the connector artifacts or files as we don't want to imply trademark or copyright. Please remove all the copyright content from all the connector files.

Reply:
@sriyen-msft Thanks for the Information. I have modified the REAdME.md accordigly